knowledge-base/styling-filtering-ui-distinct-values-datagrid-maui.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Styling the Filtering UI with Distinct Values - DataGrid for MAUI
+description: Learn how to style the Filtering UI in the DataGrid for MAUI by changing the color of the are with the checkboxes.
```

Comment:
Something is wrong with the description. `of the are` is probably `of the area`?